charts/tikv-importer/templates/tikv-importer-service.yaml

```diff
@@ -3,17 +3,18 @@ kind: Service
 metadata:
   name: {{ .Values.clusterName }}-importer
```

Comment:
I think we only need to change the service name from `{{ .Values.clusterName }}-importer` to `{{ .Release.Name }}-importer`, no need to change the other labels or selectors, please help confirm.

Reply:
we use `__meta_kubernetes_pod_label_app_kubernetes_io_name` (and some other labels and annotations) in the Prometheus config. so we need to change the labels.

Reply:
I don’t think we need to change the existing label of helm, we can add a cluster label. `app.kubernetes.io/name` and `app.kubernetes.io/instance` represents the container iteself.

Reply:
I think the issue here is that the value of instance label is `app.kubernetes.io/instance: {{ .Release.Name }}`, which is used to construct the service name in TidbMonitor controller, however, the service name deployed by the chart is `{{ .Values.clusterName }}-importer`, and they do not match.

Reply:
If we not spec clustername fied , it will use release name , right? How to deal with it?

Reply:
> If we don't spec clustername field , it will use release name , right?

NO, and I have removed `if not specified, the chart release name will be used` in this PR.

if don't specify this filed, an error will be reported, like:

```
Error: unable to build kubernetes objects from release manifest: error validating "": error validating data: unknown object type "nil" in ConfigMap.metadata.labels.app.kubernetes.io/instance
```